<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRecipeWindow</class>
 <widget class="QDialog" name="AddRecipeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>808</height>
   </rect>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Apply to QLabel (example) */
QLabel {
    color: #333333; /* Text color */
    font-family: &quot;Clothe PERSONAL USE ONLY&quot;; /* Font family */
    font-size: 58px; /* Font size */
    font-weight: normal; /* Font weight (normal, bold, etc.) */
    text-align: left; /* Text alignment */
    padding: 5px; /* Padding around text */
}

/* Apply to all widgets */
* {
    /* Global text color */
    color: #333333;
    /* Global font */
    font-family: Arial, sans-serif;
    font-size: 12px;
}

/* Example of specific text styling based on widget state */
QLabel:disabled {
    color: #999999; /* Color for disabled state */
    font-style: italic; /* Italicize text for disabled state */
}

</string>
   </property>
   <property name="text">
    <string>Add a Recipe</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Apply to QLabel (example) */
QLabel {
    color: #333333; /* Text color */
    font-family: &quot;Clothe PERSONAL USE ONLY&quot;; /* Font family */
    font-size: 20px; /* Font size */
    font-weight: normal; /* Font weight (normal, bold, etc.) */
    text-align: left; /* Text alignment */
    padding: 5px; /* Padding around text */
}

/* Apply to all widgets */
* {
    /* Global text color */
    color: #333333;
    /* Global font */
    font-family: Arial, sans-serif;
    font-size: 12px;
}

/* Example of specific text styling based on widget state */
QLabel:disabled {
    color: #999999; /* Color for disabled state */
    font-style: italic; /* Italicize text for disabled state */
}

</string>
   </property>
   <property name="text">
    <string>Name of the Dish</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="ingredientsTextEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>231</width>
     <height>371</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="typeLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Apply to QLabel (example) */
QLabel {
    color: #333333; /* Text color */
    font-family: &quot;Clothe PERSONAL USE ONLY&quot;; /* Font family */
    font-size: 20px; /* Font size */
    font-weight: normal; /* Font weight (normal, bold, etc.) */
    text-align: left; /* Text alignment */
    padding: 5px; /* Padding around text */
}

/* Apply to all widgets */
* {
    /* Global text color */
    color: #333333;
    /* Global font */
    font-family: Arial, sans-serif;
    font-size: 12px;
}

/* Example of specific text styling based on widget state */
QLabel:disabled {
    color: #999999; /* Color for disabled state */
    font-style: italic; /* Italicize text for disabled state */
}

</string>
   </property>
   <property name="text">
    <string>Type of the Dish</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeComboBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="cuisineLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Apply to QLabel (example) */
QLabel {
    color: #333333; /* Text color */
    font-family: &quot;Clothe PERSONAL USE ONLY&quot;; /* Font family */
    font-size: 20px; /* Font size */
    font-weight: normal; /* Font weight (normal, bold, etc.) */
    text-align: left; /* Text alignment */
    padding: 5px; /* Padding around text */
}

/* Apply to all widgets */
* {
    /* Global text color */
    color: #333333;
    /* Global font */
    font-family: Arial, sans-serif;
    font-size: 12px;
}

/* Example of specific text styling based on widget state */
QLabel:disabled {
    color: #999999; /* Color for disabled state */
    font-style: italic; /* Italicize text for disabled state */
}

</string>
   </property>
   <property name="text">
    <string>Cuisine</string>
   </property>
  </widget>
  <widget class="QLabel" name="tagsLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Apply to QLabel (example) */
QLabel {
    color: #333333; /* Text color */
    font-family: &quot;Clothe PERSONAL USE ONLY&quot;; /* Font family */
    font-size: 20px; /* Font size */
    font-weight: normal; /* Font weight (normal, bold, etc.) */
    text-align: left; /* Text alignment */
    padding: 5px; /* Padding around text */
}

/* Apply to all widgets */
* {
    /* Global text color */
    color: #333333;
    /* Global font */
    font-family: Arial, sans-serif;
    font-size: 12px;
}

/* Example of specific text styling based on widget state */
QLabel:disabled {
    color: #999999; /* Color for disabled state */
    font-style: italic; /* Italicize text for disabled state */
}

</string>
   </property>
   <property name="text">
    <string>Tags (Commas separated)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>591</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ingredientsLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Apply to QLabel (example) */
QLabel {
    color: #333333; /* Text color */
    font-family: &quot;Clothe PERSONAL USE ONLY&quot;; /* Font family */
    font-size: 20px; /* Font size */
    font-weight: normal; /* Font weight (normal, bold, etc.) */
    text-align: left; /* Text alignment */
    padding: 5px; /* Padding around text */
}

/* Apply to all widgets */
* {
    /* Global text color */
    color: #333333;
    /* Global font */
    font-family: Arial, sans-serif;
    font-size: 12px;
}

/* Example of specific text styling based on widget state */
QLabel:disabled {
    color: #999999; /* Color for disabled state */
    font-style: italic; /* Italicize text for disabled state */
}

</string>
   </property>
   <property name="text">
    <string>Ingredients</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tagsLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>591</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="instructionsLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>320</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Apply to QLabel (example) */
QLabel {
    color: #333333; /* Text color */
    font-family: &quot;Clothe PERSONAL USE ONLY&quot;; /* Font family */
    font-size: 20px; /* Font size */
    font-weight: normal; /* Font weight (normal, bold, etc.) */
    text-align: left; /* Text alignment */
    padding: 5px; /* Padding around text */
}

/* Apply to all widgets */
* {
    /* Global text color */
    color: #333333;
    /* Global font */
    font-family: Arial, sans-serif;
    font-size: 12px;
}

/* Example of specific text styling based on widget state */
QLabel:disabled {
    color: #999999; /* Color for disabled state */
    font-style: italic; /* Italicize text for disabled state */
}

</string>
   </property>
   <property name="text">
    <string>Instructions</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="instructionsTextEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>331</width>
     <height>371</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>730</y>
     <width>681</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 50px;  /* Adjust the radius to make the button round */
    background-color: #000000;  /* Example background color */
    color: white;
    border: none;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    font-size: 16px;
    margin: 4px 2px;
}

QPushButton:hover {
    background-color: #45a049;  /* Background color when hovering */
}
</string>
    </property>
    <property name="text">
     <string>Add Recipe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 50px;  /* Adjust the radius to make the button round */
    background-color: #000000;  /* Example background color */
    color: white;
    border: none;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    font-size: 10px;
}

QPushButton:hover {
    background-color: #45a049;  /* Background color when hovering */
}
</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="cuisineLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>591</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
